README.md

```diff
@@ -1 +1,164 @@
-# Final-capstone-backend
\ No newline at end of file
+# Final-capstone-backend
```

Comment:
# Frontend review

- [ ] When  I start the app, I get an infinite requests to the server loop:

![image](https://github.com/monicah79/Final-capstone-backend/assets/57421823/6efce4b9-5a5f-415f-99d8-168ebd12fb3d)

Also, the rails console gets crazy trying to get "laptops", hundreds of times.

## Tips and potential fixes

This is the problem:

![image](https://github.com/monicah79/Final-capstone-backend/assets/57421823/79f0f190-61a4-4695-8901-21edc4908730)

Since your app is shipping with zero laptops, that `useEffect` will fire infinitely as long as there isn't at least one laptop. 

This also means all your new users will run into this.

You could:

1. Include some laptops (at least 1) in your seeds.rb file. (Recommended)
2. Hardcode at least one laptop in the frontend.
3. Debug that useEffect so it only runs `once` regardless of the case. (Recommended too).

Kindly, fix this requests loop so your users (and the server) have a better experience and your app runs as intended. 💪 

- [ ] When I create a reservation, I immediately get redirected to "my reservations", but this happens:

![image](https://github.com/monicah79/Final-capstone-backend/assets/57421823/f353717b-4710-408f-ad50-1cbf113c0fdb)

When I reload the page, it works.

This is because the reservation worked, and was recorded in the database, but the frontend variables didn't have the chance to update with the new data, and as a result, I get a blank page.

Kindly fix this bug so the app works as intended. 👍 

- [ ]  In App.jsx please, kindly refactor the main `App` component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the rest of the functions (I found 7)

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/monicah79/Final-capstone-backend/assets/57421823/fa8e809b-fd6e-400e-9b19-e5f61339457e)


That way, your code will meet the latest standards. 💪

- [ ] Kindly, remove the 2 console.log statements found in `reservationSlice.js`:

![image](https://github.com/monicah79/Final-capstone-backend/assets/57421823/ccf09455-fb57-4865-8008-a8bad3069b58)

That way, your repo's cleanliness will be improved. 👍